visual_behavior/data_access/loading.py

```diff
@@ -287,6 +300,27 @@ def get_filtered_behavior_session_table(release_data_only=True):
     return behavior_sessions
 
 
+def get_second_release_candidates():
```

Comment:
Could you add a short doc string please to summarize what this is supposed to do?

Reply:
Yes, good suggestion!

Reply:
i am going to revise this function soon, so i just added a doc string saying so :)